presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -33,6 +34,9 @@ namespace facebook::presto {
 static constexpr size_t kTaskPeriodGlobalCounters{2'000'000}; // 2 seconds.
 // Every two seconds we export memory counters.
 static constexpr size_t kMemoryPeriodGlobalCounters{2'000'000}; // 2 seconds.
+// Every two seconds we export exchange source counters.
+static constexpr size_t kExchangeSourcePeriodGlobalCounters{
+    2'000'000}; // 2 seconds.
 // Every 1 minute we clean old tasks.
 static constexpr size_t kTaskPeriodCleanOldTasks{60'000'000}; // 60 seconds.
 // Every 1 minute we export cache counters.
```

Comment:
I start to think these are not necessary to have different names. But I'm going to do another refactor on this PeriodicTaskManager so I'll do them together.

Reply:
I think memory related counters can use the same time interval.